triton-client/src/selectors.ts

```diff
@@ -0,0 +1,15 @@
+import { createSelector } from "@reduxjs/toolkit"
+import { RequestState } from "./store/requests"
+import { RootState } from "./store/store"
+
+
+
+const selectRequestById = (state: RootState) => state.requestsState.requestById
```

Comment:
Can this lead confusion between the request own ID and the datasetID it actually uses ?

Reply:
Yes.

Reply:
Actually no. I would normally denote explicitly that it is datasetID.